### PR TITLE
[12.0][ADD]hr_gamification_email_notification

### PR DESCRIPTION
**Pull request by Khalid-SerpentCS**
HR Gamification Email Notification
==============================
This module Sends Email to all the Employees with all the award given in
the last week.

### CODE REVIEW

**Comment by alexey-pelykh** [comment]
@sbidoul ^ OCA bot gone crazy :D

**Comment by etobella** [comment]
Why OSI?

**Comment by sbidoul** [comment]
The manifest of this module is broken in several ways.  But okay, I guess the bot could survive to that :)

**Comment by Khalid-SerpentCS** [comment]
> Why OSI?

Solved!

File: hr_gamification_email_notification/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+{
+    "name": "HR Gamification Email Notification",
+    "version": "12.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
+    "summary": """Send Email to all the Employees with all the award given in
+     the last week.""",
+    "category": "Helpdesk",
+    "maintainers": ["Khalid-SerpentCS"],
+    "website": "http://www.opensourceintegrators.com",
```
Comment (bodedra) [inline hr_gamification_email_notification/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/hr",
```

**Review by bodedra — COMMENTED** [review]
@Khalid-SerpentCS Would you please add a README folder and move manual content from README.rst file.

File: hr_gamification_email_notification/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "HR Gamification Email Notification",
+    "version": "12.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
+    "summary": """Send Email to all the Employees with all the award given in
+     the last week.""",
+    "category": "Helpdesk",
```
Comment (bodedra) [inline hr_gamification_email_notification/__manifest__.py]:
```suggestion
    "category": "Human Resources",
```

**Review by bodedra — COMMENTED** [review]
For the better visibility.

File: hr_gamification_email_notification/data/badge.xml [hunk 3]
```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (max3903) [inline hr_gamification_email_notification/data/badge.xml]: @Khalid-SerpentCS Can you rename the file with the model of the data it contains `mail_template.xml`?

Comment (max3903) [inline hr_gamification_email_notification/data/badge.xml]:
```suggestion
```

File: hr_gamification_email_notification/data/badge.xml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+<odoo>
```
Comment (max3903) [inline hr_gamification_email_notification/data/badge.xml]:
```suggestion
<odoo noupdate="1">
```

File: hr_gamification_email_notification/__init__.py [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2020 Open Source Integrators
```
Comment (max3903) [inline hr_gamification_email_notification/__init__.py:1]:
```suggestion
# Copyright (C) 2020 Open Source Integrators
# Copyright (C) 2020 Serpent Consulting Services
```

File: hr_gamification_email_notification/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2020 - TODAY, Open Source Integrators
```
Comment (max3903) [inline hr_gamification_email_notification/__manifest__.py:1]:
```suggestion
# Copyright (C) 2020 Open Source Integrators
# Copyright (C) 2020 Serpent Consulting Services
```

File: hr_gamification_email_notification/__manifest__.py [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "HR Gamification Email Notification",
+    "version": "12.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
```
Comment (max3903) [inline hr_gamification_email_notification/__manifest__.py]:
```suggestion
    "author": "Open Source Integrators, Serpent Consulting Services, Odoo Community Association (OCA)",
```

File: hr_gamification_email_notification/data/badge.xml [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+                    <p>
+                    Dear Employee,<br/>
+                    Below are the award given in the last week.
+                    </p>
+                        <table width="900" border="1" cellpadding="2" style="min-width: 800px; background-color: white; border-collapse:collapse;">
+                            <thead style="background-color: blue;color: white;text-align: left;">
+                                <tr>
+                                    <th>Badge</th>
+                                    <th width="350">Comment</th>
+                                    <th>Awarded Date</th>
```
Comment (max3903) [inline hr_gamification_email_notification/data/badge.xml]:
```suggestion
                                    <th>Awarded On</th>
```

File: hr_gamification_email_notification/__manifest__.py [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+     the last week.""",
+    "category": "Human Resources",
+    "maintainers": ["Khalid-SerpentCS"],
+    "website": "https://github.com/OCA/hr",
+    "depends": ["hr_gamification"],
+    "data": ["data/ir_cron_data.xml", "data/badge.xml", "views/badge.xml"],
+    "qweb": [],
+    "auto_install": False,
+    "application": False,
+    "installable": True,
```
Comment (max3903) [inline hr_gamification_email_notification/__manifest__.py:26]:
```suggestion
    "installable": True,
    "development_status": "Stable",
```

File: hr_gamification_email_notification/readme/CONFIGURE.rst [hunk 10]
```diff
@@ -0,0 +1,2 @@
+* Basic data configuration.
+
```
Comment (max3903) [inline hr_gamification_email_notification/readme/CONFIGURE.rst]: Please remove the file.

File: hr_gamification_email_notification/readme/CONTRIBUTORS.rst [hunk 11]
```diff
@@ -0,0 +1,2 @@
+* Maxime Chambreuil <mchambreuil@opensourceintegrators.com>
+* Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>
```
Comment (max3903) [inline hr_gamification_email_notification/readme/CONTRIBUTORS.rst]:
```suggestion
* Mohammed Khalid <m.khalid@serpentcs.com>
```

File: hr_gamification_email_notification/readme/DESCRIPTION.rst [hunk 12]
```diff
@@ -0,0 +1,2 @@
+This module Sends Email to all the Employees with all the award given in
+the last week.
```
Comment (max3903) [inline hr_gamification_email_notification/readme/DESCRIPTION.rst]:
```suggestion
This module sends an email every week to all the employees with all the badges awarded during
the previous week.
```

File: hr_gamification_email_notification/readme/USAGE.rst [hunk 13]
```diff
@@ -0,0 +1 @@
+
```
Comment (max3903) [inline hr_gamification_email_notification/readme/USAGE.rst]:
```suggestion
* Go to *Settings > Email > Templates*
* Search for "Received badges" to review/adjust the email content
```

File: hr_gamification_email_notification/data/badge.xml [hunk 14]
```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="email_template_badge_weekly_awards" model="mail.template">
+            <field name="name">Received Badge</field>
```
Comment (max3903) [inline hr_gamification_email_notification/data/badge.xml]:
```suggestion
            <field name="name">Received Badges</field>
```

**Review by bodedra — COMMENTED** [review]
LG. :+1: Please squash your commit.

**Comment by pedrobaeza** [comment]
Or maybe make it a circle 😝

**Review by bodedra — APPROVED** [review]

**Comment by dreispt** [comment]
Hello, I did some tests on this, it it wasn't working - the message was created, but I didn't see it in the General channel. 
I need to dig deeper to find fix suggestion.

**Comment by dreispt** [comment]
@Khalid-SerpentCS Can you attend my last comment, please?

**Comment by Khalid-SerpentCS** [comment]
> Hello, I did some tests on this, it it wasn't working - the message was created, but I didn't see it in the General channel.
> I need to dig deeper to find fix suggestion.

Hi Daniel, Mail is generated. But not going to general channel. You can find on Technical>Settings>Emails. Thanks

**Review by marcelsavegnago — APPROVED** [review]

**Pull request closed without merging**